rpc/src/v1/impls/eth.rs

```diff
@@ -180,9 +180,9 @@ impl<C, SN: ?Sized, S: ?Sized, M, EM, T: StateInfo + 'static> EthClient<C, SN, S
 	fn rich_block(&self, id: BlockNumberOrId, include_txs: bool) -> Result<Option<RichBlock>> {
 		let client = &self.client;
 
-		let client_query = |id| (client.block(id), client.block_total_difficulty(id), client.block_extra_info(id));
+		let client_query = |id, is_pending| (client.block(id), client.block_total_difficulty(id), client.block_extra_info(id), is_pending);
```

Comment:
Just inline `false` here, no point to pass `is_pending`

Reply:
Ooops right.